### PR TITLE
Add a bit more description for community

### PR DESCRIPTION
**Pull request by ernoma**
I think there should be more detailed description for the community. It is also Hacktoberfest, https://hacktoberfest.digitalocean.com/ ;) There are also social impact projects: https://www.hotosm.org/updates/hacktoberfest-a-call-for-contributing-to-social-impact-projects/.

### CODE REVIEW

**Comment by Abbe98** [comment]
@ernoma this is awesome thanks! Images is key in this type of application's README.

Just a minor change request from my side. It would be great if the links where in Markdown. The common behavior in Github README's is that links open in the current window anyway. 

Have a great Hacktoberfest :smiley: :jack_o_lantern:

File: README.md [hunk 1]
```diff
@@ -4,9 +4,30 @@ A web application for browsing and learning sign languages, powered by Wikidata.
...
+
+Once you have run the above commands, open the <a href="http://localhost:8080/sign-language-browser/" target="_blank">http://localhost:8080/sign-language-browser/</a> page.
+
+You will see the following UI:
+
+<img src="docs/front_page.png" width="676">
+
+It lists the languages that have at least one item (can be thought as a term) stored in the <a href="https://www.wikidata.org/" target="_blank">www.wikidata.org</a> site with a link to a sign language video. When you click the language, you will get a list of the items with the linked sign language videos:
+
+<img src="docs/word_list.png" width="676">
```
Comment (Abbe98) [inline README.md]: Images too in markdown btw

**Review by Abbe98 — CHANGES_REQUESTED** [review]
Markown for links and images.

**Comment by ernoma** [comment]
Alright, the links and images are now in Markdown. I think GitHub Markdown doesn't support image resizing, so I resized the images.

**Comment by Abbe98** [comment]
@ernoma Thank you very much for the contribution!

**Pull request merged**